content/guides/ecommerce-connection/step0-who-is-your-connector-for.md

```diff
@@ -0,0 +1,53 @@
+# Who is your connector for?
+
+## Personnas
+
+You will find below user archetypes we use to help you guide your decisions about product features, interactions, and even visual design of your connector. By designing for the archetype — whose goals and behavior patterns are well understood — you can satisfy the broader group of people represented by that archetype.
+
+|![Julia](../../img/personas/Julia.png)|![Peter](../../img/personas/Peter.png)|![Filips](../../img/personas/Filips.png)|
+|:------------------------------------:|:------------------------------------:|:----------------------------------------------:|
+|     **Julia:** Product Manager       |        **Peter:** IT Manager         |    **Filips:** Julia/Peter's Integrator |
+
+
+## Personnas' needs
+
+You will find below the main needs, associated to each user profile, in terms of process, documentation, actions related to your future eCommerce connector.
+
+### Julia's needs
+
+::: info
+Julia needs an easy way to export her PIM data into her eCommerce solution.
+:::
+
+So she needs:
+
+* An **easy to follow process** to import automatically her PIM data into the eCommerce solution.
+
+* An **easy to follow strategy** to define the periodicity of these automatic imports.
+
+* A **friendly Connector UI** to trigger manually the connector if needed (full or partial imports)
+
+* A **friendly UI** to log all automatic and manual import processes and a notification system to be alerted when something goes wrong.
+
+### Peter and/or Filips' needs
+
+::: info
+Peter and/or Filips needs a easy way to install, configure and maintain your connector.
+:::
+
+So they need:
+
+* A **download/installation process:** need an easy to follow process for downloading and installing the connector.
```

Comment:
I don't understand this sentence, feels like the "need an..." is not needed!

Reply:
I will change this sentence.